src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -230,7 +230,7 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
       width: '600px'
     }
 
-    let fileDeleteRef = this.dialog.open(DeleteFileModal, fileDeleteConfig);
+    let fileDeleteRef:MatDialogRef<DeleteFileModal> = this.dialog.open(DeleteFileModal, fileDeleteConfig);
```

Comment:
added type reference to MatDialogRef<DeleteFileModal>
to be able to compile fileDeleteRef.componentInstance.onDelete with typescript
otherwise it was throwing unknown function/property error

Did same for other modals